### ISSUE TITLE
Set up brabners on OI file sharing

### ISSUE THREAD

**Issue by luke-strange** [open-innovations/true-north#30]
Can we set up brabners as a user on our file stash to share some of their data with us? Let me know if you need anything else to do this @gilesdring .

Once this is done, open below as new issue:
- [ ] build pipeline to process this true north list data.

**Comment by luke-strange** [comment]
Done and shared with them. Will send email to remind them.
